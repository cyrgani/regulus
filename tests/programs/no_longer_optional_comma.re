# previously accepted, errors now
_(print("a") print("b"))
