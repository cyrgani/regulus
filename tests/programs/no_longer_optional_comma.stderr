
SyntaxError: missing comma in argument list
at programs/no_longer_optional_comma.re:2:14